<launch>
	<arg name="loggerPath" value="/home/ubuntu/Poseidon/www/webroot/record/"/>
	<node pkg="logger_text" name="logger_text" type="logger_text_node" output="screen" args="$(arg loggerPath)"/>
	<test test-name="test_logger_service_node" pkg="logger_service" type="test_logger_service_node" />
</launch>
